migrations/2020/Version20200410190351.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20200410190351 extends AbstractMigration
+{
+    public function up(Schema $schema) : void
+    {
+        $this->addSql('ALTER TABLE `pa_payment` ADD `closed_by` varchar(64) NULL AFTER `dateClosed`;');
+        $this->addSql('UPDATE `pa_payment` SET closed_by = "unknown" WHERE transaction_payer IS NULL');
```

Comment:
```suggestion
        $this->addSql('UPDATE `pa_payment` SET closed_by = "Neznámý uživatel" WHERE transaction_payer IS NULL AND state = \'completed\'');
```

Reply:
btw. u MySQL jde string zavírat do uvozovek?

Reply:
Nebo jako uživatele mít *Hospodaření* jako u historických mailů? https://github.com/skaut/Skautske-hospodareni/pull/1226/files#diff-552763942bd538e9d96974331684f39cR34

Reply:
upravil jsem to na `Hospodaření` a v uvozovkach mi to vpohodě funguje